packages/core/src/login/webview/vue/backend.ts

```diff
@@ -34,6 +34,7 @@ import { AuthSSOServer } from '../../../auth/sso/server'
 import { getLogger } from '../../../shared/logger/logger'
 
 export abstract class CommonAuthWebview extends VueWebview {
+    private readonly className = 'CommonAuthWebview'
```

Comment:
It looks like some of the time you have the className declared inside of the class and then others outside of the class. Which one should we standardize on?